src/items/itemDialog.component.test.tsx

```diff
@@ -228,7 +220,7 @@ describe('ItemDialog', () => {
       createView();
 
       await modifyDetailsValues({
-        usageStatus: 'Used',
+        usageStatus: 'U{arrowdown}{arrowdown}{enter}',
```

Comment:
```suggestion
        # In Use
        usageStatus: 'U{arrowdown}{arrowdown}{enter}',
```
Why does this one require two arrowdown's and the others only one? Is it worth putting a comment on what this is selecting? Same below.

Reply:
Alot of the tests have two arrowdowns as well - that selected `Used`. To be honest I don't know why I put some as just one arrowdown, I can make them all consistent? When it doesn't matter what the usage status is

Reply:
I think its worth doing it just for this file, I think they all for values that used to be `usageStatus: 'Used'`, but now it is selecting 'In Use' instead.

Reply:
@MatteoGuarnaccia5 Sorry could you remove the 2nd arrow down instead of adding them to the rest